docs/Team/Operations.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Operations"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# The Day to Day
+
+## Resources
+As a company, we have a tendency to throw people in the deep end and see if they can swim. Our business is fast-paced so the traits that allow someone to thrive in that kind of situation are necessary, but we can at least take the alligators out of the deep-end.
+
+### Where Stuff Goes and Where to Find It Again
+We have different places where documents and notes can go. Rather than just throwing stuff on the floor like my 7 year old, let's organize it.
+
+#### The Guidelines
+You are here. The Guidelines site serves a few purposes:
+
+* It's a public repository for how we do things
+* It demonstrates what we do and how
+* It helps with onboarding
+
+So, if a document fits into any of those categories, then add it and publish it. The final published product should be high quality and have gone through the [pull-request and code review process](http://localhost:4000/Code#code-review) so that multiple sets of eyes have seen it.
+
+#### The Shared Google Drive
+We have our own space in Google Drive, [TW Dev](https://drive.google.com/drive/folders/0B4f4uAcTHp4ZfjU4b29mN3VsRFhEOXBBRG5wZXBLbV9mY2d6OGZfTUpRS09HWWMxMjc2Um8?usp=sharing). Google Docs and apps are a great way to collaborate on documentation and other writings that are not quite appropriate for publishing here on The Guidelines. Some examples include:
+
+* Guides or other information commonly shared with clients.
+* Documentation about processes that haven't been published to Guidelines, yet. (Once these have been published to Guidelines, the document should be archived in Drive.)
+* Internal documentation, such as client information that should not be shared.
+* Slide decks from presentations given at meetings, meetups, and conferences.
+* Running lists for meeting notes. 
+* Silly things like how each person pronounces 'GIF' and who prefers emacs over vim.
+
+You are empowered to add documents here, but it is up to all of us to keep it organized. This folder is shared with the whole dev team, but is findable by the company at large, so mind the permissions tab on the spreadsheet where we detail the weaknesses of upper management and how to exploit them.
```

Comment:
i have negative connotations with the word "empowered," as it is an oft-abused buzzword. however, it actually is used very fittingly in this sentence, so if i'm the only one that has a negative gut reaction to the word i'm good with keeping it.

Reply:
It is overused here... how about "encouraged"?

Reply:
that was my first thought also